test/iron-fit-behavior.html

```diff
@@ -500,6 +500,17 @@
             assert.equal(rect.height, elRect.height, 'no cropping');
           });
 
+          test('negative verticalOffset does not crop element', function() {
+            // Push to the bottom of the screen.
+            parent.style.top = (window.innerHeight - 50) +'px';
+            el.verticalAlign = 'top';
+            el.verticalOffset = -10;
+            el.refit();
+            var rect = el.getBoundingClientRect();
+            assert.equal(rect.top, window.innerHeight - 60, 'top ok');
+            assert.equal(rect.bottom, window.innerHeight, 'bottom ok');
```

Comment:
This line fails with the previous implementation, because the element would be rendered outside the window (it's maxHeight would be too much)